### PR TITLE
Fix tests running on osx + remove auto-gen where doesnt make sense

### LINKED ISSUE #1280: Several issues with the REPL tests
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

From discord by @madhur-tandon 

```
I am kinda fixing tests related to py-repl on the synclink branch since they are flaky and have lots of timing issues.
owing to that, I randomly tried a test called test_run_clears_previous_output
according it the title, there should be only 1 cell, since we clear the previous one and run it again
but using the --dev, it seems like there are 2 cells
This can also be seen in other related tests like test_python_exception_after_previous_output and test_hide_previous_error_after_successful_run
```

Opened this issue to investigate this problem and make it better.
Even though I don't want to spend a lot of energy on this because Jeff has a great PR on the way: https://github.com/pyscript/pyscript/pull/1106
That will introduce NB behavior and completely change this and I'd rather work to that PR to get merged...
But still, will give it a try here to hopefully help Madhur.

### What browsers are you seeing the problem on? (if applicable)

_No response_

### Console info

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by marimeireles**
## Description

In the end the problem was simply a osx vs other os problem.
The `autogen` fix comes from me not paying attention in the past while changing the autogen functionality. Sorry for that!

fixes #1280

## Checklist

-   [x] All tests pass locally
-   [x] I have updated `docs/changelog.md`
-   [ ] I have created documentation for this(if applicable)

### CODE REVIEW

**Comment by marimeireles** [comment]
@madhur-tandon finally ready.
Minor change in the end. Thanks Antonio!
and thanks for catching it.
Can you also please review it?
Cheers.

File: docs/changelog.md [hunk 1]
```diff
@@ -15,11 +15,13 @@ Bug fixes
 - Fixed an issue where `pyscript` would not be available when using the minified version of PyScript. ([#1054](https://github.com/pyscript/pyscript/pull/1054))
 - Fixed missing closing tag when rendering an image with `display`. ([#1058](https://github.com/pyscript/pyscript/pull/1058))
 - Fixed a bug where Python plugins methods were being executed twice. ([#1064](https://github.com/pyscript/pyscript/pull/1064))
+- Fixes [#1280](https://github.com/pyscript/pyscript/issues/1280).
```
Comment (FabioRosado) [inline docs/changelog.md:18]: Do we want to explain it more?

I think these additions shouldn't be added to the previous release 😄  @WebReflection  created a new entry for the future release

File: docs/changelog.md [hunk 2]
```diff
@@ -15,11 +15,13 @@ Bug fixes
...
 
 Enhancements
 ------------
 - When adding a `py-` attribute to an element but didn't added an `id` attribute, PyScript will now generate a random ID for the element instead of throwing an error which caused the splash screen to not shutdown. ([#1122](https://github.com/pyscript/pyscript/pull/1122))
 - You can now disable the splashscreen by setting `enabled = false` in your `py-config` under the `[splashscreen]` configuration section. ([#1138](https://github.com/pyscript/pyscript/pull/1138))
+- Adds support for osx on repl tests.
```
Comment (FabioRosado) [inline docs/changelog.md:24]: See above

File: pyscriptjs/tests/integration/test_py_repl.py [hunk 3]
```diff
@@ -8,7 +10,11 @@ def _replace(self, py_repl, newcode):
         WARNING: this assumes that the textbox has already the focus
         """
         # clear the editor, write new code
-        self.page.keyboard.press("Control+A")
+        if "macOS" in platform.platform():
+            self.page.keyboard.press("Meta+A")
+        else:
+            self.page.keyboard.press("Control+A")
+
         self.page.keyboard.press("Backspace")
```
Comment (FabioRosado) [inline pyscriptjs/tests/integration/test_py_repl.py:18]: Oh nice!

Comment (madhur-tandon) [reply pyscriptjs/tests/integration/test_py_repl.py:18]: lol, so this was the issue 😅

**Review by madhur-tandon — APPROVED** [review]
LGTM

File: pyscriptjs/tests/integration/test_py_repl.py (hunk 3 above)
Comment (marimeireles) [reply pyscriptjs/tests/integration/test_py_repl.py:18]: aahrrr ahah
I knoww!!

**Comment by marimeireles** [comment]
Thanks @madhur-tandon
I'll find the right changelog and make the changes fabio proposed
Cheers

**Comment by madhur-tandon** [comment]
@marimeireles Let me know when you merge, I will update my `synclink` branch then

**Review by antocuni — COMMENTED** [review]
Late review, but good job :muscle: !
LGTM

**Pull request merged**